includes/content-distribution/class-admin.php

```diff
@@ -7,11 +7,12 @@
 
 namespace Newspack_Network\Content_Distribution;
 
-use Newspack_Network\Site_Role;
-use Newspack_Network\Hub\Nodes;
-use Newspack_Network\Hub\Node;
 use Newspack\Data_Events;
 use Newspack_Network\Admin as Network_Admin;
+use Newspack_Network\Content_Distribution as Content_Distribution_Class;
```

Comment:
We've not been using an alias for this in other files, should we start doing so? I thin it'd be good to standardize this.

Reply:
I agree. It's super confusing naming without it, so that's why I aliased it, but I'm up for either. Do you have a preference?

Reply:
I'm ok with the alias, do you mind doing the same for other files?

Reply:
Not at all. Will do and push to this one 👍